fw/LocalStorage.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+const PERSISTANT = "PersistantSettings";
```

Comment:
Is settings the only thing we'll have to store? Could we make this module able to store anything?